### PR TITLE
Pull in delivery config

### PR DESCRIPTION
**Pull request by jaym**

### CODE REVIEW

File: .delivery/build-cookbook/libraries/_helper.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  when 'acceptance'
+    'union'
+  when 'union'
+    'rehearsal'
+  when 'rehearsal'
+    'delivered'
+  when 'delivered'
+    nil
+  end
+end
```
Comment (schisamo) [inline .delivery/build-cookbook/libraries/_helper.rb]: Should this function (and `prev_stage`) be in some global cookbook (or native to Delivery in some way)?

**Comment by schisamo** [comment]
So we should __not__ merge this using GitHub correct? I believe it should be reviewed as a changeset on delivery.chef.co.

File: .delivery/build-cookbook/libraries/_helper.rb (hunk 1 above)
Comment (jaym) [inline .delivery/build-cookbook/libraries/_helper.rb]: this is a @cwebberOps question

**Comment by jaym** [comment]
@cwebberOps said it's cool to merge

**Pull request merged**